### PR TITLE
Add PPP to ramaven.md

### LINKED ISSUE #302: [DOC] Rama Draft PPP

### PR DESCRIPTION
**Pull request by ramaven**
### Summary

Add PPP to ramaven.md

Resolves #302 

### Checks

- [x] Link PR to issue
- [x] Add Reviewer
- [x] Pass CI

### CODE REVIEW

File: docs/team/ramaven.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    * What it does: Takes the input time string from the user and parses it
+      to create a Time object
+    * Justification: This is needed as we our ```Activity``` class has start times and end times
+      which are of the type Time.  
+    * Notable PRs: [PR #110](https://github.com/AY2122S1-CS2103T-T15-1/tp/pull/110)
+
+* **Project Management**: 
+  * Opened issues that sought to fix bugs
+    * Examples: [Issue #184](https://github.com/AY2122S1-CS2103T-T15-1/tp/issues/184)
+  * Reviewed and approved some team members' PRs for merging.
```
Comment (simonjulianl) [inline docs/team/ramaven.md:64]: You might wanna put some examples of those PRs

File: docs/team/ramaven.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        
+    * Developer guide
+      * Wrote the [find command implementation](https://github.com/AY2122S1-CS2103T-T15-1/tp/pull/286)
+
+
+* **UI Enhancement**: Change the dark theme of GoMedic to a light theme
+    * What it does: Enables the app to follow the mockup more closely by changing from the 
+      original dark grey theme of AB3 to a light theme with hues of blue and turquoise
+    * Notable PRs: [PR #120](https://github.com/AY2122S1-CS2103T-T15-1/tp/pull/120)
+    * Credits: This heavily uses JavaFx.
```
Comment (simonjulianl) [inline docs/team/ramaven.md:42]:
```suggestion
    * Credits: This heavily uses CSS feature of JavaFx. 
```

File: docs/team/ramaven.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+  * What it does: Allows user to search for entries based on their category (Doctor, Patient, or Activity),
+    and specify a field to which the user input will be matched to. 
+  * Justification: Allows for more targeted search so users can get the specific entries
+    that they are looking for
+  * Highlights: For example, within the patient category, users can search specifically
+    within the blood type category. 
+  * Notable PRs: [PR #160](https://github.com/AY2122S1-CS2103T-T15-1/tp/pull/160)
+  * Credits: SE-EDU AB3 for creating the project template
+    
+* **New feature**: Develop datetime parser
```
Comment (simonjulianl) [inline docs/team/ramaven.md:54]: Erm, sorry about this but I replace the parser using localDateTime function which takes like 6 lines. So yeah, you might want to rephrase this as it is not there anymore (like initial skeleton of datetime parser or sth along this line)

**Review by simonjulianl — CHANGES_REQUESTED** [review]
Can help to check my comments

**Comment by ramaven** [comment]
> Can help to check my comments

Yup done

File: docs/team/ramaven.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
+---
```
Comment (simonjulianl) [inline docs/team/ramaven.md:1]: wait why do you need two PRs for this ?

**Review by simonjulianl — APPROVED** [review]
LGTM btw, and you wanna close the other pr
